### PR TITLE
Microverse Portfolio project: finish mobile version

### PR DESCRIPTION
**Pull request by Fombi-Favour**
In this activity, we:
- Updated the index.html file
- Update the style.css file
- Completed the mobile version

### CODE REVIEW

**Review by Johnadibe — COMMENTED** [review]
Hi @Fombi-Favour,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that does not pass linter check for CSS. Please make sure that all linter checks are passed.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by V-Blaze — CHANGES_REQUESTED** [review]
Hi @Fombi-Favour  , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

dcentl8-9b96e129-9725-401b-ac34-2feb990202a0

## Invalid Code Review Request

## You have submitted a project that has some of the important sections of the readme file missing. According to Microverse all the sections of your readme file must be present and filled with content before requesting for review. Currently, your readme file does not have the `Future Feature` section, Please include this in your readme file and also add at least 2 features that could be added to your project in the future. :zap: 
- [ ] I'll also point out here that the link to the license page in your readme is broken, Please look into this too. :+1: :smile: 


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -36,5 +36,147 @@ <h5>LET'S CONNECT</h5>
...
+        </main>
+    </section>
+    <footer class="contact-me">
+        <main class="shapes">
+            <h1>Contact me</h1>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
+            <input type="text" class="input-col">
+            <input type="email" class="input-col">
+            <textarea class="input-col comment" placeholder="Write your message here..."></textarea>
+            <button type="button" class="btn-detail contact">Get in touch</button>
```
Comment (Alfredbis29) [inline index.html]: - [ ] I can see that you did a great job on this project, but they're still to improve on something for your project to look more professional, I can see that you forget to put a `placeholder` in your form to refer a user what to complete in the form, kindly add it for a good user experience,  kindly fix this :point_down: don't forget to follow figma guidelines, make sure you put the same `font` as for Figma

Your design
Screenshot from 2023-04-28 00-03-06

Figma Design
Screenshot from 2023-04-27 23-56-38

File: README.md [hunk 2]
```diff
@@ -115,6 +115,14 @@ You can deploy this project on github following instructions here:
...
 
+## 🔭 Future Features <a name="future-features"></a>
+
+-  **Add details project page with live demo and viewing of source code**
+-  **Add Desktop platform features**
+-  **Customization of styles of the two platforms**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
 ## 🤝 Contributing <a name="contributing"></a>
```
Comment (Alfredbis29) [inline README.md:123]: - [ ] I noticed that one of the links you included, specifically the M.I.T license link, is broken and no longer functional.
As you know, links are an essential part of any project, and it's crucial to ensure that they are functional and lead to the intended content. In this case, the broken link may hinder the ability of others to access important information related to your project. Kindly fix this :point_down: 
Screenshot from 2023-04-28 00-11-25

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
#  STATUS: REQUIRED CHANGES 
 ## Hi @Fombi-Favour   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  
 ## Highlights 
 :heavy_check_mark: Linters are passing.
:heavy_check_mark: You have a good code structure.
:heavy_check_mark: Linters are green
 
 ## Required Changes 

- [ ] Check Under the review

 ## Optional suggestions 

- [ ] Kindly remove all comments from the template in your `README` , it makes your  project looks unprofessional :wink: 

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
 
 Cheers and Happy coding! :clap: :clap: :clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @Fombi-Favour 👋, 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Descriptions are descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

As part of professional [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
- Let's always use descriptive commit messages - each commit message should be meaningful and small enough to describe it in one sentence. 🛠

 Here are some examples of descriptive commits:

descriptive_commits_example


Here are some commits that are not descriptive:

not_descriptive_commits

IMPORTANT: As it is not easy to change commit history, kindly correct this behavior in the future. 🛠💪

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**